Documentation/operations/upgrade-warning.rst

```diff
@@ -4,6 +4,12 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. warning::
```

Comment:
Looking at the [preview](https://deploy-preview-17755--cilium-doc.netlify.app/operations/upgrade.html#upgrade-guide), I think we would ideally integrate this text into the guide somewhere a bit more naturally rather than as the first warning that appears on the page. While this is an important point, I don't think that it needs to be the first thing on the page and it probably doesn't need to be called out in a warning box.  The upgrade guide has barely started at this point and this would mean there are already two warnings :thinking:

Reply:
Done. Curious how did you get the preview line? (I tested the change locally via `make render-docs`)

Reply:
The link? We haven't had this for too long, but if you scroll down to the bottom in the checks, there is one called `deploy/netlify`. If you click on "Details" there then it'll provide a preview of the PR docs.